### ISSUE TITLE
Have website be navigable with tab for the purpose of accessibility

### ISSUE THREAD

**Issue by davidmetraux** [fspoettel/arkham.build#436]
Areas not accessible with tab:
Card/Investigator Lists
Can't access the Recommendations, Notes, Tools Tabs

Why does this matter?
Better for people that require screen readers or can't use a mouse for whatever reason.

Further steps:
Add "skip link" so people could skips part of the website while navigating with tab.

**Comment by fspoettel** [comment]
Card lists and tabs are accessible via arrow keys.

For card lists, keyboard navigation is available when the search is focused, use arrow keys and enter to open a card modal.

**Comment by davidmetraux** [comment]
Nice!

**Comment by fspoettel** [comment]
closing this as I think the current accessibility via tab + arrow is enough. we cannot actually use tab for the card lists due to the virtualisation approach we are using so changing arrow -> tab is not possible. feel free to reopen if you disagree and want more to happen here.
